2주차/12789/12789_kotlin_페퍼.kt

```diff
@@ -0,0 +1,33 @@
+import java.util.*
+
+fun main() {
+    val n = readln().toInt()
+    val numbers = readln().split(" ").map { it.toInt() }.toMutableList()
+    val stack = Stack<Int>()
+
+    var target = 1
+    while (numbers.isNotEmpty()) {
+        if (stack.isNotEmpty() && stack.peek() == target) {
+            target++
+            stack.pop()
+            continue
+        }
```

Comment:
```suggestion
        while (stack.isNotEmpty() && stack.peek() == target) {
            stack.pop()
            target++
        }
```
이렇게 해도 괜찮아보이는데, 혹시 이렇게 하지 않은 이유가 있나요?

Reply:
저도 케이의견과 같아요!